src/main/java/com/manufacturing/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,45 @@
+package com.manufacturing.web.filters;
+
+import com.manufacturing.lib.Injector;
+import com.manufacturing.service.DriverService;
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("com.manufacturing");
+    private final DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+    }
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) request;
+        HttpServletResponse resp = (HttpServletResponse) response;
+        String servletPath = req.getServletPath();
+        if (servletPath.equals("/login") || servletPath.equals("/drivers/create")) {
+            chain.doFilter(req, resp);
+            return;
```

Comment:
it's optional but ideally, we can replace it with Set, so this construction will be more readable when the number of URLs grows.